code/game/machinery/ai_slipper.dm

```diff
@@ -4,133 +4,66 @@
 	icon_state = "liquid_dispenser"
```

Comment:
Should this default to `liquid_dispenser_on` now

Reply:
It should only be showing the "on" sprite if the device has power and was turned on, so I thought it was best to have it default to off and let the proc update it when the game loads.

Reply:
Fair enough